src/services/blocks/BlocksTraceService.ts

```diff
@@ -0,0 +1,102 @@
+import { BlockHash } from '@polkadot/types/interfaces';
+import { InternalServerError } from 'http-errors';
+
+import {
+	BlocksTrace,
+	BlocksTraceOperations,
+} from '../../types/responses/BlocksTrace';
+import { AbstractService } from '../AbstractService';
+import { BlockTraceResponse, isBlockTrace, isTraceError, Trace } from './trace';
+
+const DEFAULT_TARGETS = 'pallet,frame,state';
+// :extrinsic_index & frame_system::Account
+const DEFAULT_KEYS =
+	'3a65787472696e7369635f696e646578,26aa394eea5630e07c48ae0c9558cef7b99d880ec681799c0cf30e8886371da9';
+const UNEXPECTED_RPC_RESPONSE = 'Unexpected response to state_traceBlock RPC';
```

Comment:
Just double checking but should it be `state_trace_block`, just looks a little weird but good chance I am wrong here?

Reply:
resolved, i understand this is what it is now.